scraper/Requirements.txt

```diff
@@ -0,0 +1,12 @@
+certifi==2021.10.8
```

Comment:
A lot of the stuff here has nothing to do with the project, certifi? Did you just run pip freeze on your entire system? 

This project needs like max websockets and that's about it.

Reply:
You've got requests which by default includes a lot of those other things. Same for websocket, has a few dependencies

Did you mean to use websocket-client? That has fewer dependencies, but is not what was utilized here.

Reply:
[requirements justifications.txt](https://github.com/ProstoSanja/place-2022/files/8417655/requirements.justifications.txt)

Reply:
Ah, i apologise, im not really familiar with pythion, venvs, so thought it only needed a list of direct dependencies, and not entire tree. Just checked locally, all good.